src/main/resources/rules/ConstructionRobotAssistanceDecisionRules.fcl

```diff
@@ -0,0 +1,53 @@
+FUNCTION_BLOCK ConstructionRobotAssistanceDecision
+
+VAR_INPUT
+    timeRobotIsBlocked : REAL;
+    robotBatteryStatus : REAL;
+    waitingTimeForHuman : REAL;
+END_VAR
+
+VAR_OUTPUT
+    suggestedWorkSharingApproach : REAL;
+END_VAR
+
+FUZZIFY timeRobotIsBlocked     // in minutes (0 to 15 minutes)
+    TERM SHORT := (0, 1) (4, 1) (6, 0);
+    TERM MODERATE := (4, 0) (6, 1) (9, 1) (11, 0);
+    TERM LONG := (9, 0) (11, 1) (15, 1);
+END_FUZZIFY
+
+FUZZIFY robotBatteryStatus    // in percentage (0 to 100 percentage)
+    TERM LOW := (0, 1) (20, 1) (30, 0);
+    TERM MEDIUM := (20, 0) (30, 1) (60, 1) (70, 0);
```

Comment:
Just a general idea that came to mind, to be discussed: 
- we could maybe simplify the rules (and thus make them easier to understand) by considering only `LOW` or `not LOW` for the battery status, i.e. medium/high would not be relevant. 
- If we would decide to do so, I am wondering: would it be sufficient to just define _one_ membership function `LOW` and remove both other membership functions? Because the rules could then just use `LOW` and `NOT LOW`.